src/Tribe/Aggregator/Migrate.php

```diff
@@ -245,6 +245,30 @@ public function is_ical_migrated() {
 	}
 
 	/**
+	 * Filters out empty values
+	 *
+	 * NOTE: we aren't using array_filter because EVEN with an empty() alias, the results are
+	 * unpredictable
+	 *
+	 * @param array $original_values
+	 *
+	 * @return array
+	 */
+	public function filter_out_unwanted_values( $original_values ) {
```

Comment:
Yeah, something along this vein may have a nice life in tribe-common - but I don't think this iteration as-is is as flexible as we could make it. In the interest of moving to Smoketest, I'm going to leave this here and puzzle over a better tribe-common method at a later date.